docs/tutorial/part_2.rst

```diff
@@ -92,13 +92,13 @@ Add the imports of ``mark_safe`` and ``json`` and add the ``room`` view function
 Create the route for the room view in ``chat/urls.py``::
 
     # chat/urls.py
-    from django.conf.urls import url
+    from django.urls import path
     
     from . import views
     
     urlpatterns = [
-        url(r'^$', views.index, name='index'),
-        url(r'^(?P<room_name>[^/]+)/$', views.room, name='room'),
+        path('chat',views.index, name='index'),
+        path('chat/<str:room_name>/',views.room,name='room')
```

Comment:
Is it normal to change the url here ?

Before: / -> views.index, room_name/ -> views.index
After: chat/ -> views.index, chat/room_name/ -> views.room

Reply:
> Is it normal to change the url here ?
> 
> Before: / -> views.index, room_name/ -> views.index
> After: chat/ -> views.index, chat/room_name/ -> views.room

yes it is normal to change url here, for example, if we enter http://127.0.0.1:8000/chat/lobby/, it will take pattern `chat/lobby/` and will give views.room